pkg/cmd/pulumi/new.go

```diff
@@ -813,6 +861,102 @@ func saveConfig(stack backend.Stack, c config.Map) error {
 	return saveProjectStack(stack, ps)
 }
 
+func runtimeOptions(ctx *plugin.Context, info *workspace.ProjectRuntimeInfo,
```

Comment:
```suggestion
func promptRuntimeOptions(ctx *plugin.Context, info *workspace.ProjectRuntimeInfo,
```

I think would be a little more descriptive.  Very minor nit though, so feel free to ignore if you disagree.